plugins/org.komodo.shell/resources/version.properties

```diff
@@ -0,0 +1 @@
+komodo.version=${project.parent.version}
```

Comment:
This file, `version.properties`, should be added to the `build.properties` file.